starwars/src/components/StarWarsList.js

```diff
@@ -20,20 +20,18 @@ export default function StarWarsList() {
 
     return (
         <Container>
-            <Row>
-                {chars.map(char => {
-                    return (
-                        <StarWarsCard
-                            key={char.height}
-                            name={char.name}
-                            birth={char.birth_year}
-                            gender={char.gender}
-                            hair={char.hair_color}
-                            eye={char.eye_color}
-                        />
-                    )
-                })}
-            </Row>
+            {chars.map(char => {
+                return (
+                    <StarWarsCard
+                        key={char.height}
+                        name={char.name}
+                        birth={char.birth_year}
+                        gender={char.gender}
+                        hair={char.hair_color}
+                        eye={char.eye_color}
+                    />
+                )
+            })}
```

Comment:
Good job using the key property, although try to use a value for key that is more unique, such as an id or the index of the array that is being mapped. If two char objects have the same height value, then the keys will be the same and that negates the purpose of using the key property.